ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -555,7 +555,9 @@ private NamedBlobRecord run_get_v2(String accountName, String containerName, Str
       statement.setInt(1, accountId);
       statement.setInt(2, containerId);
       statement.setString(3, blobName);
-      try (ResultSet resultSet = statement.executeQuery()) {
+      logger.debug("Getting blob name from MySql. Query {}", statement);
+      ResultSet resultSet = statement.executeQuery();
+      try {
         if (!resultSet.next()) {
```

Comment:
If you just want to log for debug purpose. you can just log it like below, why need to move the resources outside of try() block?

```
try (ResultSet resultSet = statement.executeQuery()) {
logger.debug(.....);
}
```

Reply:
Sure. I was also trying to print the mysql query in catch clause. Due to that, I moved it outside. But, added back the try-with-resources now